dbcon.php

```diff
@@ -2,7 +2,7 @@
 //---- Includes db parameters.
 include('db.php');
 
-
+// Test by SAM
 //--- Sample class to create connection using PDO
 class db {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename class "db" to match the regular expression ^[A-Z][a-zA-Z0-9]*$. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=php%3AS101)